editor/components/block-list/insertion-point.js

```diff
@@ -92,13 +92,13 @@ export default compose(
 		);
 
 		return {
-			templateLock: getEditorSettings().templateLock,
+			locking: getLockedState( insertionPoint.rootUID ),
```

Comment:
Prop `locking` should also have its name updated in accordance with `getLockedState`, _or_ should be renamed entirely to something like `isLocked` with `isLocked: !! getLockedState( … )`

Reply:
Naming was updated.

Reply:
- At the editor level, we name template locking `templateLock`
- At the block level, we name template locking `lock`